NulpAppApi/Controller/FavouritesTableViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  FavouritesTableViewController.swift
+//  NulpAppApi
+//
+//  Created by Ivan Sapiga on 10/15/18.
+//  Copyright © 2018 Ivan Sapiga. All rights reserved.
+//
+
+import UIKit
+
+class FavouritesTableViewController: UITableViewController {
+    let placesArray = FavouriteManager.getFavourites()
```

Comment:
This is not an MVP approach. Correct way is to create presenter instance for this ViewController and then tell him when view is shown/hidden. Then presenter tells what items should be rendered